config/locales/publishers/build_reference_forms.yml

```
@@ -0,0 +1,241 @@
+            general_attitude:
+              blank: Rate the candidate's general attitude
+            technical_competence:
+              blank: Rate the candidate's technical competence (if relevant)
+            leadership:
+              blank: Rate the candidate's leadership skills (if relevant)
+        referees/referee_details_form:
+          attributes:
+            name:
+              blank: Enter your name
+            job_title:
+              blank: Enter your job title
+            phone_number:
+              blank: Enter your phone number
+            email:
+              blank: Enter your email address
+              invalid: Enter a valid email address in the correct format, like name@example.com
+            organisation:
+              blank: Enter the name of your organisation
+            complete_and_accurate:
+              accepted: Confirm that this form is correct to the best of my knowledge
+
+  helpers:
+    hint:
+      referees_employment_reference_form:
+        employment_start_date: For example, 27 3 2007
+    label:
+      referees_can_give_reference_form:
+        can_give_reference_options:
+          true: Yes, I can give them a reference
+          false: No, I am unable to give them a reference
+      referees_can_share_reference_form:
+        is_reference_sharable_options:
+          true: Yes, if they request it
+          false: No, it should be treated as confidential
+
+      referees_referee_details_form:
+        name: Name
+        job_title: Job title
+        phone_number: Phone number
+        email: Email address
+        organisation: Organisation name
+        complete_and_accurate_options:
+          "1": I confirm the answers given in this form have been provided in good faith and are correct to the best of my knowledge and belief.
+
+      referees_employment_reference_form:
+        how_do_you_know_the_candidate: How long have you known the candidate and in what capacity?
+        currently_employed_options:
+          true: "Yes"
+          false: "No"
+        would_reemploy_current_options:
+          true: "Yes"
+          false: "No"
+        would_reemploy_any_options:
+          true: "Yes"
+          false: "No"
+        reason_for_leaving: Reason for leaving/considering leaving your employment
+        would_reemploy_current_reason: Please give reasons for you answer
+        would_reemploy_any_reason: Please give reasons for you answer
+
+      referees_reference_information_form:
+        under_investigation_options:
+          true: "Yes"
+          false: "No"
+        warnings_options:
+          true: "Yes"
+          false: "No"
+        allegations_options:
+          true: "Yes"
+          false: "No"
+        not_fit_to_practice_options:
+          true: "Yes"
+          false: "No"
+        able_to_undertake_role_options:
+          true: "Yes"
+          false: "No"
+
+      referees_how_would_you_rate_form:
+        punctuality_options:
```

Suggestion if we want to keep the option definitions in a single place:
```suggestion
        punctuality_options: &rating_options
```